### PR TITLE
Read symbols from a mobile style sample

### PR DESCRIPTION
**Pull request by ThadT**
https://devtopia.esri.com/runtime/common-samples/issues/1275

@nCastle1 I updated the version of Xamarin forms to fix an exception I was running into: https://github.com/xamarin/xamarin.forms/issues/5188

The UI for this sample was a little difficult. Xamarin Forms looks good on UWP but is a little rough on iOS and Android (I'm open to suggestions!).

### CODE REVIEW

File: src/iOS/Xamarin.iOS/Samples/Symbology/SymbolsFromMobileStyle/SymbolsFromMobileStyle.cs [hunk 1]
```diff
@@ -0,0 +1,602 @@
...
+            // Create a list of the available colors for the symbol.
+            _colorList = new List<System.Drawing.Color>
+            {
+                System.Drawing.Color.Yellow,
+                System.Drawing.Color.LightGreen,
+                System.Drawing.Color.Pink
+            };
+
+            // Create a segmented control to display each available color as a button.
+            _colorSegments = new UISegmentedControl();
```
Comment (nCastle1) [inline src/iOS/Xamarin.iOS/Samples/Symbology/SymbolsFromMobileStyle/SymbolsFromMobileStyle.cs]: I recommend setting the text color to black for the segmented control:

```cs
var attrs = new UITextAttributes();
attrs.TextColor = UIColor.Black;
_colorSegments.SetTitleTextAttributes(attrs, UIControlState.Normal);
```

File: src/Android/Xamarin.Android/Samples/Symbology/SymbolsFromMobileStyle/SymbolsFromMobileStyle.cs [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+
+            return listItem;
+        }
+    }
+
+    // A class to contain information about a symbol from a style.
+    public class SymbolLayerInfo
+    {
+        // A bitmap preview of the symbol.
+        public Bitmap Image { get; private set; }
```
Comment (nCastle1) [inline src/Android/Xamarin.Android/Samples/Symbology/SymbolsFromMobileStyle/SymbolsFromMobileStyle.cs]: I would check to see if these setters can be removed altogether

Comment (nCastle1) [reply src/Android/Xamarin.Android/Samples/Symbology/SymbolsFromMobileStyle/SymbolsFromMobileStyle.cs]: It looks like changing to a field caused the binding to stop working on WPF; I recommend re-adding the getters, e.g. `public WinMedia.ImageSource ImageSrc { get; }`.

Comment (nCastle1) [reply src/Android/Xamarin.Android/Samples/Symbology/SymbolsFromMobileStyle/SymbolsFromMobileStyle.cs]: Applies to UWP as well

**Comment by ThadT** [comment]
Thanks @nCastle1 
I'll fix the property accessors and confirm the binding works before merging.

**Pull request merged**